mne/preprocessing/realign.py

```diff
@@ -85,25 +87,40 @@ def realign_raw(raw, other, t_raw, t_other, verbose=None):
         f"{raw.times[-1] * dr_ms_s:0.1f} ms)"
     )
 
-    # 2. Crop start of recordings to match using the zero-order term
-    msg = f"Cropping {zero_ord:0.3f} s from the start of "
+    # 2. Crop start of recordings to match
     if zero_ord > 0:  # need to crop start of raw to match other
-        logger.info(msg + "raw")
+        logger.info(f"Cropping {zero_ord:0.3f} s from the start of raw")
         raw.crop(zero_ord, None)
         t_raw -= zero_ord
     else:  # need to crop start of other to match raw
-        logger.info(msg + "other")
-        other.crop(-zero_ord, None)
-        t_other += zero_ord
+        logger.info(f"Cropping {(zero_ord / first_ord):0.3f} s from the start of other")
+        other.crop(-(zero_ord / first_ord), None)
+        t_other += zero_ord / first_ord
```

Comment:
The same computation is done three times (this requires effort to see and retype) so it seems a little cleaner to do the DRY version
```suggestion
        t_crop = zero_ord / first_ord
        logger.info(f"Cropping {t_crop:0.3f} s from the start of other")
        other.crop(-t_crop, None)
        t_other += t_crop
```